packages/core/test/unit/html/NodeProcessor.data.js

```diff
@@ -6,15 +6,15 @@
 
 module.exports.PROCESS_PANEL_ATTRIBUTES = `
 <panel header="# Lorem ipsum" alt="_emphasized alt_">
-  Header and alt attributes should be processed and inserted under panel as internal slots and deleted.
+  Header and alt attributes should be processed and inserted under panel as slots and internal slots respectively and deleted.
```

Comment:
Is this description alright? Or should we just leave it at "slots"?

Reply:
looks good 👍

Reply:
there's a few minor conflicts with #1498

Reply:
Have resolved the conflicts!